data/tools/unit_tree/html_output.py

```diff
@@ -957,9 +957,15 @@ def clean_uval(name):
                                         sname = T(special, "name")
                                         if sname:
                                             s.append(sname)
+                                accuracy = attack.get_text_val("accuracy", default="0")
+                                parry = attack.get_text_val("parry", default="0")
+                                if accuracy != "0":
+                                    s.append(cleantext("accuracy "+accuracy+"%"))
```

Comment:
not sure if these names are somewhere translated, but that is not in [attack] at least

Reply:
#7705 is not merged yet so I wouldn't worry too much about it.

`cleantext` is actually used in the write at the end so not necessary to call it here.

Reply:
True, didnt notice. Would be good to refactor these 2 places, but for know I would leave the double escaping and not refactor.

>>> html.escape(html.escape("accuracy -1%"))
'accuracy -1%'